persist/fs/clone/cloner.go

```diff
@@ -62,16 +82,29 @@ func (c *cloner) Clone(src FileSetID, dest FileSetID, destBlocksize time.Duratio
 	}
 
 	for {
-		id, data, checksum, err := reader.Read()
+		id, tagsIter, data, checksum, err := reader.Read()
 		if err != nil {
 			if err == io.EOF {
 				break
 			}
 			return fmt.Errorf("unexpected error while reading data: %v", err)
 		}
 
+		var tags ident.Tags
```

Comment:
this snippet seems to be repeated multiple times in the diff. mind pulling into a common place? `m3x/ident/testutil` maybe?

Reply:
Sure thing.